src/otx/algo/modules/norm.py

```diff
@@ -115,10 +121,9 @@ def build_norm_layer(cfg: dict, num_features: int, postfix: int | str = "") -> t
         if norm_layer is None:
             msg = f"Cannot find {norm_layer} in {NORM_DICT.keys()} "
             raise KeyError(msg)
-    abbr = infer_abbr(norm_layer)
+    abbr = infer_abbr(norm_layer, layer_name)
```

Comment:
```suggestion
    abbr = infer_abbr(norm_layer) if layer_name not None else layer_name
```
It seems that if `layer_name` isn't `None`, then it's used as it is. If then, how about changing this line like this and not changing `infer_abbr` function?

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/3741/commits/2d5cef1ef2bbc576bebd10fd1a96e812d6ef0548